sdk/monitor/monitor-query/review/monitor-query.api.md

```diff
@@ -30,9 +30,23 @@ export const Durations: {
 
 // @public
 export enum KnownMonitorAudience {
```

Comment:
We don't have an audience that works for all of "Monitor". So, we should name this `KnownMonitorMetricsAudience` if this is only containing metrics dataplane audience.

Reply:
This will result in breaking change since this is already exposed in the SDK. So, I prefer to keep this. Later, when there is a breaking change involved, I could add this.

Reply:
Synched up offline and added comments to address this.